sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -20,18 +20,17 @@
  *     Composed of a list of sensitivity values in pre-contingency and post-contingency states.
  * </p>
  *
- * A single sensitivity analysis should return, besides its status and some stats on the
- * analysis itself, all the sensitivity values for each factor (combination of a monitored equipment or bus and a specific
- * equipment or group of equipments). The chosen sensitivity provider offers the possibility to calculate the sensitivity
+ * Sensitivity analysis is used to assess the impact of a small modification of a network variables on the value of
+ * network functions. A combination of a variable and a function is called a sensitivity factor. A sensitivity analysis returns
+ * the sensitivity values for each factor. The sensitivity analysis API offers the possibility to calculate the sensitivities
  * on a set of contingencies besides the pre-contingency state.
- * Note that the analysis is launched only once, but the solver itself modifies the matrix for each state of the network
- * to output a full set of results. In the sensitivity API, it has been allowed to provide a list of contingencies as an input,
- * which then triggers such a sensitivity analysis.
- * The full set of results consists of :
+ * The full set of results consists of:
+ *  - the list of factors
+ *  - the list of contingencies
  *  - the list of sensitivity values in pre-contingency and post-contingency states.
```

Comment:
also the list of function reference values in pre-contingency and post-contingency states